spec/abci++/abci++_client_server.md

```diff
@@ -24,11 +24,12 @@ or custom protobuf types.
 
 For more details on protobuf, see the [documentation](https://developers.google.com/protocol-buffers/docs/overview).
```

Comment:
Please take a look at the changes made in #9182 to this file and try to include them here (so we don't lose those changes when we delete the current `spec/abci` directory.
You can show the changes I am referring to like this:
```sh
git show ae1fc74f8 -- spec/abci/client-server.md
```

Reply:
Done!